target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -29,6 +29,9 @@ glinet,gl-mt300n-v2)
 	ucidef_set_led_netdev "wifi_led" "wifi" "red:wlan" "wlan0"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x1"
 	;;
+hak5,wifi-pineapple-mk7)
+	ucidef_set_led_netdev "open_wifi_led" "wifi" "blue:system" "phy0tpt"
```

Comment:
I think you will need ucidef_set_led_wlan instead of ucidef_set_led_netdev for phy0tpt trigger?

Reply:
Why not put that in DTS instead (`linux,default-trigger`)? Also, the same LED is used as the `led-running` diag which won't work. In that case, the `led-running` should be removed or... maybe it's better to use the green LED for the diag purposes, @Foxtrot?

Reply:
I was under the impression that led-boot will already overwrite the DT trigger; if that's not the case, and only led-running will be a problem, I'd just drop led-running and move the trigger to DTS.

Reply:
@adschm the `led-boot` is used only during (pre)init, after the bootup we use `led-running`. And if there is the same LED used somewhere else (like the WiFi activity/status trigger) it won't make sense - after the boot the LED will get enabled (see: https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/diag.sh#L42) but whenever you bring up or down the WiFi, its value gets changed, plus it will be misleading for the user to see WiFi dedicated LED enabled with the WiFi itself disabled at the same time.
We restore DTS trigger for `led-boot` only (thus, you can use the same LED for `led-boot` and later in system, for whatever trigger type but it has to be configured in DTS), see: https://github.com/openwrt/openwrt/commit/a422b171ac17edde296b48e2b3e9c9845d773609

So, if the same LED is used for WiFi and diag purposes, it shouldn't be used also as the `led-running` indicator. Another story here is the fact that this board has 3 LEDs without strictly defined functions (like labeled on the device). So it's up to @Foxtrot to decide about their function. IMHO, green LED could be used for diag purposes only (so for all `led-*` in DTS) and the blue one for WiFi activity/status.

Reply:
I'd rather drop `led-running` from DTS than change it to the green LED, if possible, to avoid changing LED behavior compared to the factory firmware in a way that is very noticeable to the end user, and then use the blue LED for WiFi purposes in DTS using `linux,default-trigger = "phy0tpt";`.

What do you think?

Reply:
@Foxtrot 

> What do you think?

It's really up to you. My suggestion here came from the fact that OpenWrt has Wi-Fi disabled by default, so in that case, after first flashing the user might not be able to tell whether the board is up and running or not (the blue LED will blink during the bootup and then goes off, until the Wi-Fi is turned on).